src/angular-dragdrop.js

```diff
@@ -173,7 +177,7 @@ var jqyoui = angular.module('ngDragDrop', []).service('ngDragDropService', ['$ti
     this.move = function($fromEl, $toEl, toPos, duration, dropSettings, callback) {
       if ($fromEl.length === 0) {
         if (callback) {
-          window.setTimeout(function() {
+          $timeout(function() {
```

Comment:
Can you pass the third argument as `false` so that dirty-checking would not kick in? I think that was the whole point of using `window.setTimeout` over `$timeout`.